week-1-practise/level-1/10-function-output-3.js

```diff
@@ -1,5 +1,6 @@
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together
+  return firstWord.concat(secondWord).concat(thirdWord);
 }
```

Comment:
nice work. In fact you can call concat just once - by providing several parameters to it https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat e.g. 
`return firstWord.concat(secondWord, thirdWord);`